docs/running-in-production-dockerized.md

```diff
@@ -0,0 +1,88 @@
+# Running in Production Dockerized
+
+As the PrairieLearn source code is publicly-available, it's possible to run PrairieLearn on your own infrastructure. Running a single instance of the PrairieLearn server may be appropriate for tens or hundreds of total users, and a number of universities have done this successfully.

```

Comment:
Rather than repeating this here, I'd encourage you to leave this in the main setup page and just cross-link these two pages.

Reply:
Remove repeated section